### DISCUSSION TITLE
teamtalk web: making a web application / website with webrtc with teamtalk functionality

### DISCUSSION THREAD

**Discussion by denizsincar29** [BearWare/TeamTalk5#1736] [Ideas]
Hello! I was thinking to do a web teamtalk app on sdk, but it needs additional server to relay webrtc signals to the teamtalk sdk, to a teamtalk server and back and it will be a big delay. is there a way to make teamtalk web application with webrtc that connects straight to teamtalk? it is great because no need to install teamtalk, just sent a url to a friend and he joined the tt server on the web.

**Comment by bear101** [comment]
No, it's not possible. TeamTalk is not compatible with WebRTC.

**Comment by Sergiaws** [comment]
I also consider necesary a web client for TeamTalk, it would be easier for devices where you cannot install or use the app.
